app/src/main/java/org/p2p/wallet/infrastructure/network/interceptor/MoonpayRequestException.kt

```diff
@@ -1,13 +1,14 @@
 package org.p2p.wallet.infrastructure.network.interceptor
 
+import okio.IOException
 import org.p2p.wallet.infrastructure.network.moonpay.MoonpayErrorResponseType
 import javax.net.ssl.HttpsURLConnection
 
 class MoonpayRequestException(
     val httpCode: Int,
     val errorType: MoonpayErrorResponseType,
     override val message: String
-) : Throwable(message) {
+) : IOException(message) {
```

Comment:
Why?

Reply:
Strangely, Okhttp doesn't accept correctly if you throw `Throwable`, it works okay if the exception is `IOException`
I read on the internet that any error / exception in interceptor that is not IOException - means something FATAL for OkHttp and it crashes the app.